go/exocom/src/logger/logger.go

```diff
@@ -77,7 +78,7 @@ func (l *Logger) Messages(message structs.Message, originalName string, internal
 			text += fmt.Sprintf("--[ %s ]-[ %s ]->", originalName, internalMessageName)
 		}
 		text += "  " + receiver
-		if message.ResponseTo != "" {
+		if message.ResponseTime > time.Nanosecond {
```

Comment:
The Duration type is technically just a wrapper for the int64 type, so the default is zero. Changed.